docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -123,28 +126,12 @@ You will see test logs similar to the following:
 
 A clean pass of the above unit tests is an indication that your Open Enclave setup was successful.
 
-You can start playing with the Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
-
 For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) document.
 
-## Installing the SDK on the local machine
-
-To install the debug SDK on the local machine use the following:
-
-```powershell
-cd openenclave/build/x64-Debug
-cmake -DCMAKE_INSTALL_PREFIX=C:/openenclave ../..
-ninja install
-```
-
-This installs the SDK in `C:/openenclave`, the path specified for
-`CMAKE_INSTALL_PREFIX`. This install path is assumed for the rest of the
-instructions.
-
 ## Build and run samples
 
-To build and run the samples, please look [here](/samples/README_Windows.md).
+To build and run the samples independently of SDK building/installation, please look [here](/samples/README_Windows.md).
```

Comment:
Dave has previously suggested not linking non-descriptive words like "here" or "this". Consider instead:
```suggestion
To build and run the samples without building and then installing the OE SDK, please refer to the [README for Windows samples](/samples/README_Windows.md).
```